lips/lip-1.mdx

```diff
@@ -11,92 +11,62 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
-
-**reference_id**: Every Object type contains a `reference_id` field which is a unique reference ID for the Object.  The reference ID is used as a unique identifier of the Object. The reference ID is always specified by the payment initiator VASP (the VASP which originally created the Object). This value should be unique, and formatted as “<creator_vasp_onchain_address_bech32>_<unique_id>”. For example, ”lbr1x23456abcd_seqABCD“. As an example, see the reference_id field on the [PaymentObject](#paymentobject). Unicode utf-8 encoded.
-
-# Overall Off-Chain Properties
-The Off-Chain Protocol formulates methodologies to provide the following high-level properties:
-
-### Channel Establishment
-The Off-Chain Protocol allows two entities A and B to form a communication Channel for manipulating a consistent database of Objects and agreeing on Commands that manipulate those Objects.
-
-### Command Issuance By Either Party
-Both ends of the Channel can issue Commands upon Shared Objects.
-
-### Object Versioning
-To ensure a consistent view of Object states, every Object is versioned. Mutations may only occur upon the latest Version of an Object.
+**Shared Object**: All Objects contained within a Command - for example `PaymentObject`s, are considered as "Shared Objects" - meaning that either VASP may create a new Command to modify the Object, and will do so during the typical life-cycle of an Object - an example being the addition of required data within the `KycDataObject` from both VASPs to a payment Object. Only one VASPs in a Channel can may at any time attempt to initiate and execute a Commands on Shared Objects, defined by the state of the Object.
 
-### Concurrency Control
-The Off-Chain Protocol manages concurrent requests on Objects and ensures that Object states will not become out of sync.
+**Version**: The state of an Object at a specific point in time. Upon creation an Object is assigned version 0. Every creation or mutation increases the Version of the Object which was created/mutated by one.  VASPs operate upon the latest Version of Shared Objects.  Once a Version is mutated, it is no longer a valid Version upon which a new Command may depend - the newly updated value of the Version is the latest Version for that Object and must be acted upon.
 
-### Eventual Consistency
-The Off-Chain Protocol ensures eventual consistency of Shared Objects.
-
-# Basic Off-Chain Building Blocks
+**reference_id**: Every Object type contains a `reference_id` field which is a unique reference ID for the Object. The reference ID is always specified by the payment initiator VASP (the VASP which originally created the Object). This value should be unique. We recommend the hexadecimal representation of a fresh >16 bytes random byte-string as a way of creating such values. The field is Unicode utf-8 encoded.
```

Comment:
I want to bring up this question again: 
What are the benefits of allowing utf-8 charsets v.s plain and simple alphanumeric? 
Using utf-8 charsets complicates databases (e.g. MySQL) operations and needs extra caution to deal with regard SQL injection.

Reply:
Do you guys think it makes sense to add regex to reference_id (and other fields)? I think allowing _any_ character for reference id is certainly a bad thing

Reply:
could we consider UUID only?

Reply:
if someone put in sequence: 1, 2, 3 from their db index, is it ok? when 2 vasps do this, it's a problem right?

Reply:
We should probably say that unique identifier of a payment is (sender, receiver, ref_id), so different vasp don't need to worry about clash with other vasps

Reply:
Good point @andll !

Reply:
(sender, receiver, ref_id) as unique identifier won't work if you reference this in another payment object, it will be very confusing to figure out what is sender and receiver  of the refered id